docs/extend-graphql-schema.md

```diff
@@ -3,4 +3,568 @@ id: extend-graphql-schema
 title: Extend the GraphQL Schema
 ---
 
-How to extend the existing GraphQL Schema?
\ No newline at end of file
+How to extend the existing GraphQL Schema?
+
+You might at some point need to expose new data in your GraphQL endpoint, to
+support new features. Front-Commerce allows you to extend and override any part
+of the initial schema by leveraging features from the GraphQL Schema Definition
+Language (<abbr title="Schema Definition Language">SDL</abbr>).
```

Comment:
Imo, this introduction could be improved. I feel like it misses a link between the two sentences.

Suggestion :
> One might at some point need to expose new data in your GraphQL endpoint, to support new features. We can achieve this in Front-Commerce by leveraging features from the GraphQL Schema Definition Language (<abbr title="Schema Definition Language">SDL</abbr>).
>
> Specifically, we will need to create and register our own custom « modules » on the server. Under the hood, Front-Commerce Lite uses [GrAMPS](https://gramps.js.org/) datasources, and Front-Commerce uses a similar approach with additional features.